actors/builtin/storage_miner/storage_miner_actor.go

```diff
@@ -334,12 +334,15 @@ func (a *StorageMinerActor) ExtendSectorExpiration(rt Runtime, sectorNumber abi.
 	return &vmr.EmptyReturn{}
 }
 
-func (a *StorageMinerActor) TerminateSector(rt Runtime, sectorNumber abi.SectorNumber) *vmr.EmptyReturn {
+func (a *StorageMinerActor) TerminateSectors(rt Runtime, sectorNumbers []abi.SectorNumber) *vmr.EmptyReturn {
```

Comment:
Should we consider RLE+ for this parameter?

Reply:
possibly, lets resolve this as a follow up too.